app/src/main/java/com/myvetpath/myvetpath/ViewSubsActivity.java

```diff
@@ -0,0 +1,168 @@
+            mDataset = myDataset;
+            mDatesset = myDatesset;
+            mCaseset = myCaseset;
+            clickListener = listener;
+        }
+
+        // Create new views (invoked by the layout manager)
+        @Override
+        public SubsAdapter.MyViewHolder onCreateViewHolder(ViewGroup parent,
+                                                         int viewType) {
+            // create a new view
+            View v = LayoutInflater.from(parent.getContext())
+                    .inflate(R.layout.subrow, parent, false);
+            final MyViewHolder myViewHolder = new MyViewHolder(v);
+            v.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View view) {
+                    clickListener.onSubClick(view, myViewHolder.getAdapterPosition());
+                }
+            });
+            return myViewHolder;
+        }
+
+        // Replace the contents of a view (invoked by the layout manager)
+        @Override
+        public void onBindViewHolder(MyViewHolder holder, int position) {
+            // - get element from your dataset at this position
+            // - replace the contents of the view with that element
+            holder.titleTextView.setText(mDataset[position]);
+            holder.dateTextView.setText(mDatesset[position]);
+            holder.caseTextView.setText(mCaseset[position]);
+
+        }
+
+        @Override
+        public int getItemCount() {
+            return mDataset.length;
+        }
+
+    }
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_view_subs);
+        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
+        toolbar.setTitle("Submissions");
+        setSupportActionBar(toolbar);
+
+        create_sub_activity = new Intent(this, CreateSubActivity.class);
+        sub_details_activity = new Intent(this, SubDetailsActivity.class);
+
+        mRecyclerView = (RecyclerView) findViewById(R.id.subsRecyclerView);
+
+        subsTitles = getResources().getStringArray(R.array.subsTitles);
+        subsDates = getResources().getStringArray(R.array.subsDates);
+        subsCaseID = getResources().getStringArray(R.array.subsCaseIDs);
+
+        mAdapter = new SubsAdapter(subsTitles, subsDates, subsCaseID, new CustomSubClickListener() {
+            @Override
+            public void onSubClick(View v, int position) {
+                sub_details_activity.putExtra("pos", position);
+                startActivity(sub_details_activity);
+            }
+        });
+        mLayoutManager = new LinearLayoutManager(this);
+
+        mRecyclerView.setLayoutManager(mLayoutManager);
+        mRecyclerView.setAdapter(mAdapter);
+
+        FloatingActionButton fab = (FloatingActionButton) findViewById(R.id.fab);
+        fab.setOnClickListener(new View.OnClickListener() {
```

Comment:
Did we want a floating action button for the view submissions screen?

Reply:
I know we talked about letting users delete submissions locally and I didn't know if we wanted a button to do this or if we wanted to add it to the toolbar for this screen. I left it in, but I can take it out easily if we decide not to use it.